src/Item/Item.router.spec.ts

```diff
@@ -1230,6 +1315,49 @@ describe('Item router', () => {
             })
           })
         })
+
+        describe("and the item's metadata (not rarity nor collection) and content is being updated", () => {
+          let currentDate: Date
+
+          beforeEach(() => {
+            currentDate = new Date()
+            jest.useFakeTimers()
+            jest.setSystemTime(currentDate)
+            const originalItemMock = { ...new Item() }
+            mockItem.mockImplementation((createdItem) => {
+              return {
+                ...originalItemMock,
+                upsert: jest.fn().mockResolvedValueOnce(createdItem),
+              } as any
+            })
+
+            mockItem.findOne.mockResolvedValueOnce(dbItem)
+          })
+
+          afterEach(() => {
+            jest.useRealTimers()
+          })
+
+          it('should respond with the updated item', async () => {
+            const response = await server
+              .put(buildURL(url))
+              .send({ item: itemToUpsert, name: 'aNewName' })
+              .set(createAuthHeaders('put', url))
+              .expect(STATUS_CODES.ok)
+
+            expect(response.body).toEqual({
+              data: {
+                ...Bridge.toFullItem(dbItem),
+                local_content_hash:
+                  'bafkreiejcahiwp257urwn5u2wn5os3mcjhvid6eqhmok7tezzhvdm5njqy',
+                eth_address: wallet.address,
+                created_at: dbItem.created_at.toISOString(),
+                updated_at: currentDate.toISOString(),
+              },
+              ok: true,
+            })
+          })
```

Comment:
Tests look good. Just a question, would it make sense to just use `expect.any(Date)` and avoid having to use fakeTimers?

I know it's less specific, but I also think that the simplicity might be worth it, relegating fakeTimers for real time sensitive tests